src/components/Cart/Cart.vue

```diff
@@ -1,13 +1,77 @@
 <template>
+  <aside v-if="!openedValue" class="np-cart--closed">
+    <img :src="userAvatar" alt="avatar" class="np-cart__avatar"/>
+    <div class="np-cart__arrow__container" @click="toggleMenu">
+      <img alt="left arrow" class="np-cart__arrow" :src="leftArrow"/>
+    </div>
+  </aside>
+
+  <aside v-else class="np-cart--opened">
+    <User/>
+    <Steps/>
+    <Summary/>
+    <div class="np-cart__arrow__container" @click=toggleMenu>
+      <img alt="right arrow" class="np-cart__arrow" :src="rightArrow"/>
+    </div>
+  </aside>
 
 </template>
 
-<script>
-export default {
-name: "Cart"
-}
+<script lang="ts">
+import {defineComponent} from "vue";
+import {mapGetters,mapMutations} from "vuex";
+import User from "./User/User.vue";
+import Steps from "./Steps/Steps.vue";
+import Summary from "./Summary/Summary.vue";
+
+const Cart = defineComponent({
+  components: {
+    User,
+    Steps,
+    Summary,
+  },
+  data() {
+    return {
+      leftArrow: 'https://www.flaticon.com/svg/vstatic/svg/271/271220.svg?token=exp=1610449680~hmac=fdfcf071159efaa5ab2fa6cde3dd1758',
+      rightArrow: 'https://www.flaticon.com/svg/vstatic/svg/271/271228.svg?token=exp=1610449680~hmac=8769db5a2be5abf094bc17d8753de6f0',
+    }
```

Comment:
Proponuję używać tzw. compositionAPI przy tworzeniu aplikacji. To jeden ze "sposobów pisania kodu" we vue. Tak jak wcześniej próbowaliście pisać w ClassAPI (z tymi dekoratorami @Props @Component itd, ale nie wyszło bo nie jest kompatybilne z Vue3 + TypeScript), to jest jeszcze compositionAPI którego dokumentacje macie tutaj: https://v3.vuejs.org/guide/composition-api-introduction.html . W firmie dużo używamy właśnie compositionAPI i warto to umieć :)

Reply:
Ostatecznie zamiast data() trzeba by było użyć setup() 
i wszystko co jest w setup() jest jak lifecycleMethod o nazwie created().

Reply:
Znika też methods i computed i wszystko ląduje w setup(),